components/inputs/test/input-number.test.js

```diff
@@ -82,6 +82,43 @@ describe('d2l-input-number', () => {
 		});
 	});
 
+	describe('invalid values', () => {
+		it('should reset value', async() => {
+			const elem = await fixture(normalFixture);
+			elem.value = 'helloworld123';
+
+			setTimeout(() => dispatchEvent(elem, 'change'));
+			await oneEvent(elem, 'change');
+
+			expect(elem.value).to.equal(undefined);
+			expect(elem._formattedValue).to.equal('');
+		});
+	});
+
+	describe('re-formatting values', () => {
```

Comment:
These 2 tests are kind of just testing that the number formatting/parsing in the `d2l-intl` library are working correctly. I'm OK verifying that, but could we just call the tests something like `it should format value using Intl` and `it should parse value using Intl`?